AudioLinkWebProject/Assets/autocorrelatorspherefield.shader

```diff
@@ -0,0 +1,196 @@
+Shader "autocorrelatorspherefield"
```

Comment:
I think we should adjust this shader or make a different one for the demo. No offense or anything, but its kind of harsh on the eyes

Reply:
make a suggestion

Reply:
Would be nice to show off more than just the autocorrelator data, I suppose. We could take some of the demo shaders we already have lying around, and make some tiles showing different data, kinda like what Orels had going for the website he was tinkering with. A small grid of quads, or something like that

Reply:
doesn't have to be in this PR

Reply:
so it's not blocking, closing this comment then